client/src/components/Public/EntitySearch/EntitySearchModal.js

```diff
@@ -69,10 +69,13 @@ const EntitySearchModal = ({
               placeholder={FIND_ENTITY_TITLE}
               value={entitySearchValue}
               onChange={setEntitySearchValue}
-              ref={(input) => {
-                const el = ReactDOM.findDOMNode(input)
-                el && el instanceof HTMLElement && el.focus()
-              }}
+              ref={(input) => setTimeout(
+                () => {
+                  const el = ReactDOM.findDOMNode(input)
```

Comment:
Please, change `ref` to `innerRef` and get rid of this `findDOMNode`. I am also not happy about that `setTimeout` hack, but I didn't find another way. Do you have any idea why `autoFocus` doesn't work?

Reply:
This code was working a while back and wasn't changed since, not really sure why it stopped. The `setTimeout` is the only thing I found that fixes it.